libs/utils/executor.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, test_env, experiments_conf=None):
         - <wload>   : the identified of one of the specified workload
         - <run_id>  : the progressive execution number from 1 up to the
                       specified iterations
+
+        After the workloads have been run, the Executor object's `experiments`
```

Comment:
Cannot this chunk be squashed in the patch which introduces this list of Experiments?